pkgs/development/tools/build-managers/gradle/default.nix

```
@@ -1,4 +1,4 @@
-{ jdk8, jdk11, jdk17 }:
+{ jdk8, jdk11, jdk17, lib }:
```

```suggestion
{ lib, jdk8, jdk11, jdk17 }:
```
lib should be at the beginning of the list.